modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/ContentStructureResourceTest.java

```diff
@@ -57,7 +65,9 @@
 
 		return _toContentStructure(
 			_addDDMStructure(
-				testDepotEntry.getGroup(), contentStructure.getName()));
+				(assetLibraryId == irrelevantDepotEntry.getDepotEntryId()) ?
```

Comment:
fiuuu. Why do you need it?

Reply:
Yes... it's ugly. It is needed to now when to use the irrelevant depot entry group or the depot entry group, as the only thing the method knows is the assetLibraryId and didn't know from which of them it is. 

Maybe if we get rid of the irrelevantDepotEntry as you suggest in another comment we can avoid this also